Gruntfile.js

```diff
@@ -51,13 +51,14 @@ module.exports = function(grunt) {
     }
   };
 
-  var prefixMatch = "\\n *(function |var |static )?";
+  var regexLF = (grunt.util.linefeed === "\r\n" ? "(\\u000D\\n)" : "\\n"); //get the correct Lf for the operating system
```

Comment:
What we are building here is a regular expression, and we need to have the string "\n" not the character \n. That's why all these lines escape the \ character ```\\```.